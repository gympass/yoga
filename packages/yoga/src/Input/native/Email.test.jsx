import React from 'react';
import { render } from '@testing-library/react-native';

import { ThemeProvider, Input } from '../..';

describe('<Input.Email />', () => {
  describe('Snapshots', () => {
    it('should match with default input', () => {
      const { toJSON } = render(
        <ThemeProvider>
          <Input.Email />
        </ThemeProvider>,
      );

      expect(toJSON()).toMatchSnapshot();
    });
  });
});
