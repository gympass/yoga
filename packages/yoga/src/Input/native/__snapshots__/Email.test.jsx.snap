// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Input.Email /> Snapshots should match with default input 1`] = `
<View
  full={false}
  style={
    [
      {
        "position": "relative",
        "width": 320,
      },
      {},
    ]
  }
>
  <TextInput
    autoCapitalize="none"
    autoCompleteType="email"
    cleanable={true}
    disabled={false}
    editable={true}
    focus={false}
    full={false}
    keyboardType="email-address"
    onBlur={[Function]}
    onFocus={[Function]}
    style={
      [
        {
          "borderColor": "#D7D7E0",
          "borderRadius": 8,
          "borderStyle": "solid",
          "borderWidth": 1,
          "color": "#6B6B78",
          "fontFamily": "Rubik",
          "fontSize": 14,
          "fontWeight": "400",
          "lineHeight": 20,
          "paddingBottom": 16,
          "paddingLeft": 16,
          "paddingRight": 52,
          "paddingTop": 16,
          "width": 320,
        },
        {
          "height": 52,
        },
      ]
    }
    textContentType="emailAddress"
    typed={false}
    value=""
  />
</View>
`;
